ev3dev.cpp

```diff
@@ -684,7 +701,7 @@ constexpr char gyro_sensor::mode_gyro_cal[];
 //~autogen
 
 gyro_sensor::gyro_sensor(address_type address) :
-  sensor(address, { ev3_gyro })
+  sensor(address, { ev3_gyro }, "ev3-uart", ev3_gyro)
```

Comment:
This is only needed on the BrickPi, right? Should this be guarded with `#ifdef EV3DEV_PLATFORM_BRICKPI`?

Reply:
Could do, although it won't hurt to set these properties explicitly on an EV3. Your call.

Reply:
Hmm... maybe I will stick in an ``#ifdef``. The EV3's processor is a bit underpowered so maybe we shouldn't do unnecessary things.